WcaOnRails/app/webpacker/components/DelegateProbations/index.jsx

```diff
@@ -62,7 +61,7 @@ function ProbationListTable({
 }
 
 export default function DelegateProbations() {
-  const [userId, setUserId] = React.useState();
+  const [user, setUser] = React.useState();
```

Comment:
I am not a big fan of the search bar holding the entire `user` or `person` model. Holding the `userId` only (as was the case previously) makes it much easier to store data. Why did you decide to change this to hold the entire model instead?

Reply:
Okay, the flow makes sense, but you've got to check other places in the code that already use this search component (if there are any at all) and see whether they rely on ID or on the whole model.

Reply:
Since this search component was introduced to React 1 week ago, I've checked through the commits in between. Also searched in the code. I confirm that there are no usage other than DelegateProbations/index.jsx.